### DISCUSSION TITLE
I cant get RTLSDR-Airband to generate good quality recordings - is anyone using it with good quality ?

### DISCUSSION THREAD

**Discussion by israelgoldstein** [rtl-airband/RTLSDR-Airband#409] [General]
Hi all

first of kudos to all involved with development of this project, very light application that allows recording multiple channels with light CPU overhead

but, I cant get this application to generate good Quality recordings, I've tried a few different times to install the application on different Computers, all running Linux.

if you have experience with setting up the config file, and can assist me, please reach out

I've created a Telegram Channel where I'm feeding the recordings to,  
https://t.me/+73FEq7pD01swZDAx
maybe by listening to it, you can identify what's wrong ?

**Comment by israelgoldstein** [comment]
Has anyone else been experiencing this issue ? Or is it just me.

I'd love to use this application as it's very low CPU usage, but the audio isn't very clear compared to using other applications (trunk recorder as example)

**Reply by R1ch1e-Lee** [reply]
sounds like some kind of noise mixed in with the signal

**Reply by israelgoldstein** [reply]
What would cause it ? too much gain ?
the PC generates them ?

but doesn't seem that rtl_airband settings will fix it ?

**Reply by R1ch1e-Lee** [reply]
I would try lowpass and highpass settings and adjust accordingly.  That should eliminate the worst of it.  I haven't used those settings too much myself.  I have used the notch setting with great success though to get rid of a PL tone that was really annoying.

**Comment by charlie-foxtrot** [comment]
There are a lot of parameters that can impact audio quality.  I would recommend using some GUI like SDR++ to visualize the spectrum / settings and see if your FFT size, center frequency, gain, offset, etc are correct
